pkg/user/tx_client.go

```diff
@@ -190,6 +211,8 @@ func (client *TxClient) SubmitPayForBlobsWithAccount(ctx context.Context, accoun
 
 // BroadcastPayForBlob signs and broadcasts a transaction to pay for blobs.
 // It does not confirm that the transaction has been committed on chain.
+// If no gas or gas price is set, it will estimate the gas and use
+// the provided gas price
```

Comment:
"the provided" seems confusing here b/c the first part of this sentence says "if no gas or gas price is set". Could it be rephrased to something else?

```suggestion
// the max effective gas price: max(localMinGasPrice, globalMinGasPrice).
```